### PR TITLE
vncviewer: Update to version 7.10.0

### LINKED ISSUE #133: vncviewer: doesn't update anymore
Error from the excavator
```text
vncviewer: 7.9.0 (scoop version is 7.0.1) autoupdate available
Autoupdating vncviewer
Searching hash for VNC-Viewer-7.9.0-Windows-32bit.exe in https://www.realvnc.com/en/connect/download/viewer/
Could not find hash in https://www.realvnc.com/en/connect/download/viewer/
Downloading VNC-Viewer-7.9.0-Windows-32bit.exe to compute hashes!
The remote server returned an error: (404) Not Found.
URL https://www.realvnc.com/download/file/viewer.files/VNC-Viewer-7.9.0-Windows-32bit.exe#/vncviewer.exe is not valid
ERROR Could not update vncviewer, hash for VNC-Viewer-7.9.0-Windows-32bit.exe failed!
```

investigation is required

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [ ] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other
## Description
- Change the download url to fit with the new one
- Remove the hash autoupdate section since it don't seems to exists/work anymore
## Related Tickets & Documents
Fixes #133
## [optional] What gif/emoji best describes this PR or how it makes you feel?
:alien: :fire: :bug:
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### File persistence to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### vncviewer

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
